app/utils/google_drive_api.py

```diff
@@ -47,13 +49,11 @@ def get_file_in_google_drive(google_api_call, file_to_upload):
                                         fields="id").execute()
 
 
-
-# def update_file(service, userfile, id): #works
-#         file_id = id
-#         file_metadata = { #if name is not specified when updating the file it will keep the same name
-#             'parents': '1mC7JhJA4WLp8pT7pb98NNgtSZPzqkGAn' #parent must match parent of folder
-#         }
-#         media = MediaFileUpload("path of file")
-#         service.files().update(body=file_metadata,
-#                                         media_body=media,
-#                                         fileId=file_id).execute()
\ No newline at end of file
+def update_file(service, userfile, file_id): #works
+        file_metadata = { #if name is not specified when updating the file it will keep the same name
+            'parents': ''  #put google folder ID inside of quotes to get file in that folder
```

Comment:
NIT:  Instead of using literal strings, can we store this folder ID in a `RESUME_FOLDER_ID` variable at the top of the file next to `SCOPE`